pkg/client/unversioned/extensions.go

```diff
@@ -94,6 +96,10 @@ func (c *ExtensionsClient) ThirdPartyResources(namespace string) ThirdPartyResou
 	return newThirdPartyResources(c, namespace)
 }
 
+func (c *ExtensionsClient) Generic(namespace string, gvk unversioned.GroupVersionKind) GenericResourceInterface {
+	return newGenericResources(c, namespace, gvk)
```

Comment:
Yeah, this doesn't belong in the extensions client. Is there a reason you can't just use the DynamicClient (other than that it doesn't exist yet?), if not maybe just create that? It looks like you're 85% there already.